dev/com.ibm.ws.transport.http/src/com/ibm/ws/genericbnf/internal/BNFHeadersImpl.java

```diff
@@ -2084,16 +2109,18 @@ private void createSingleHeader(HeaderKeys key, byte[] value, int offset, int le
      */
     private void addHeader(HeaderElement elem, boolean bFilter) {
         final HeaderKeys key = elem.getKey();
+        final String name = elem.getName();
```

Comment:
This should still be key.getName().  elem.getName() just calls key.getName(), so no reason to do the indirection.
```suggestion
        final String name = key.getName();
```

Reply:
Agree, changed.